.github/workflows/build-docker.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       uses: redhat-actions/buildah-build@v2.9
       with:
         image: backend
-        tags: ${{ github.event.release.tag_name }}
```

Comment:
As a result of these changes, `grocy/frontend` and `grocy/backend` images will be published using both the `grocy` application version tag (vX.Y.Z) and also the `grocy-docker` container version tag (vX.Y.Z-n_iteration).

For example, the next expected version of them will be:

- `backend` with tags `v3.3.1`, `v3.3.1-2`
- `frontend` with tags `v3.3.1`, `v3.3.1-2`

This allows the "current" Grocy v3.3.1 image to be pulled without knowing the `grocy-docker` iteration number.

It also allows customers to stick to a specific container iteration if they prefer.

And as a result, these allow the iteration number to be removed from the `docker-compose` file (and README) - building using that will pull the latest-available version per-Grocy-release.